site/content/community/get-involved.md

```diff
@@ -1,7 +1,9 @@
 ---
-title: Community
-type: docs
-menu: docs
+title: Get Involved !
+weight: 1
+type: community
+menu: "community"
+aliases: ["/community/"]
 ---
 
 # Community
```

Comment:
This page is a link for "agenda and past recordings in this Google doc."   It will be clear that it is a google doc... lets change to "Community Meeting Agenda and Past Recordings"

Reply:
I would be a fan of including the appropriate icons with the social links... Github, Slack and Twitter.   The icon is... iconic :)

Reply:
we have our own contributing guide we should link to: https://github.com/kudobuilder/kudo/blob/master/CONTRIBUTING.md
It is important because it has links for the CLA!

We also have a style guide:  https://github.com/kudobuilder/kudo/blob/master/STYLEGUIDE.md
that is less important 

It is likely useful to point them to the KEPs... especially since there is a KEP (K8S Enhancement Process) and KEP (Kudo Enhancement Process) .  https://github.com/kudobuilder/kudo/blob/master/keps/0001-kep-process.md

Reply:
All good suggestions, will look tomorrow